src/store.rs

```diff
@@ -170,3 +173,69 @@ impl RoomKeyInfo {
         self.inner.session_id.clone()
     }
 }
+
+/// Struct containing the bundle of secrets to fully activate a new devices for
+/// end-to-end encryption.
+#[derive(Debug)]
+#[wasm_bindgen]
+pub struct SecretsBundle {
+    pub(super) inner: matrix_sdk_crypto::types::SecretsBundle,
+}
+
+/// The backup-specific parts of a secrets bundle.
+#[derive(Debug)]
+#[wasm_bindgen(getter_with_clone)]
+pub struct BackupSecretsBundle {
+    /// The backup decryption key.
```

Comment:
Doc: What's the format? base64 or prefix + base58 (recovery key export format)?

Reply:
It's unpadded base64, just like in the `m.secret.send` event. Added it to the docs in: 94020b69dde69edf6963b8ad60d3af1b10ed18b1.